verification/curator-service/api/test/clients/aws-events-clients.test.ts

```diff
@@ -57,3 +57,30 @@ describe('putRule', () => {
             .catch((e) => expect(e.message).toContain('missing RuleArn'));
     });
 });
+
+describe('deleteRule', () => {
+    it('deletes the AWS CloudWatch Event rule via the SDK', async () => {
+        const deleteRuleSpy = jest.fn().mockResolvedValueOnce({});
+        AWSMock.mock('CloudWatchEvents', 'deleteRule', deleteRuleSpy);
+        const client = new AwsEventsClient('us-east-1');
+
+        await expect(client.deleteRule('passingRule')).resolves.not.toThrow();
+        expect(deleteRuleSpy).toHaveBeenCalledTimes(1);
+    });
+    it('throws errors from AWS', async () => {
+        const expectedError = new Error('AWS error');
+        AWSMock.mock(
+            'CloudWatchEvents',
+            'deleteRule',
+            (params: any, callback: Function) => {
+                callback(expectedError, null);
+            },
+        );
+        const client = new AwsEventsClient('us-east-1');
+
+        expect.assertions(1);
+        return client
```

Comment:
guessing you investigated this -- but we can't do an expect.toThrow here instead of expecting an assertion from the catch statement?

Reply:
We can -- either's perfectly valid javascript(tm) according to Jest (see [Promise](https://jestjs.io/docs/en/asynchronous#promises) vs. [`rejects.toThrow()`](https://jestjs.io/docs/en/asynchronous#asyncawait)). I like that the Promise approach is more declarative/flexible, but we can give toThrow a swing for the sake of slaying `catch` statements.